inductiva/simulators/simulator.py

```diff
@@ -36,6 +39,15 @@ def _setup_input_dir(self, input_dir: types.Path):
                 f"The provided path (\"{input_dir}\") is not a directory.")
         return input_dir
 
+    def _check_resource_pool(self, machine_group):
```

Comment:
Still not sure if I like this, however I will leave here my comments either way.